pdfgen/src/types/hierarchy/primitives/name.rs

```diff
@@ -15,39 +15,42 @@ use crate::types::WriteDictValue;
 /// When writing a name in a PDF file, a SOLIDUS (2Fh) (/) shall be used to introduce a name.
 /// No token delimiter (such as white-space) occurs between the SOLIDUS and the encoded name.
 /// Whitespace used as part of a name shall always be coded using the 2-digit hexadecimal notation.
-pub(crate) struct Name(&'static [u8]);
+pub(crate) struct Name<'a>(&'a [u8]);
```

Comment:
In previous comment I already mentioned this, but we could think about making the `Name` more generic.